packages/odyssey-react/src/components/FieldLabel/FieldLabel.stories.tsx

```diff
@@ -0,0 +1,62 @@
+/*!
+ * Copyright (c) 2021-present, Okta, Inc. and/or its affiliates. All rights reserved.
+ * The Okta software accompanied by this notice is provided pursuant to the Apache License, Version 2.0 (the "License.")
+ *
+ * You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0.
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *
+ * See the License for the specific language governing permissions and limitations under the License.
+ */
+
+import type { Story } from "@storybook/react";
+import React from "react";
+import FieldLabel from ".";
+import type { Props } from ".";
+
+export default {
+  title: `Components/FieldLabel`,
+  component: FieldLabel,
+  argTypes: {
+    children: { control: 'text' },
+    id: { control: 'text' },
+  },
+  parameters: {
+    controls: {
+      sort: 'requiredFirst'
```

Comment:
Consider opting into a lint rule to enforce a standard quote style. We're mixing double and single in this file and others.

Reply:
good thoughts [see my response above!](https://github.com/okta/odyssey/pull/975#discussion_r654361626)